common/components/WalletDecrypt/Mnemonic.tsx

```diff
@@ -86,18 +88,21 @@ export default class MnemonicDecrypt extends Component<Props, State> {
   };
 
   public onMnemonicChange = (e: React.SyntheticEvent<HTMLTextAreaElement>) => {
-    this.setState({ phrase: (e.target as HTMLTextAreaElement).value });
+    const phrase = (e.target as HTMLTextAreaElement).value;
+    const formattedPhrase = phrase.replace(/(\r\n|\n|\r|,)/gm," ").trim();
```

Comment:
Move regex into utility file. Would be nice to see if lodash has something like this out of the box to avoid rolling our own regex if possible.

Reply:
Unit tests would also be good for the utility function